CONTRIBUTING.rst

```diff
@@ -137,7 +137,15 @@ installed to run all of the environments. Then run::
     tox
 
 Please note that you need an active MongoDB instance running on localhost in
-order for the tests run. Also, be advised that in order to execute the
+order for the tests run. Save yourself some time and headache by creating a
+MongoDB user with the password defined in the `test_settings.py` file in the
+admin database (the pre-commit process is unforgiving if you don't want to
+commit your admin credentials but still have the file modified, which would be
+necessary for tox). If you want to run a local MongoDB instance along with an
+SSH tunnel to a remote instance, if you can, have the local use the default
+port and the remote use some other port. If you can't, fixing the tests that
+won't play nicely is probably more trouble than connecting to the remote and
+local instances one at a time. Also, be advised that in order to execute the
```

Comment:
yeah testing eve can be a hassle :-) Been wanting to refactor/rewrite them since forever, basically.